src/action/macos/create_fstab_entry.rs

```diff
@@ -144,64 +84,39 @@ impl Action for CreateFstabEntry {
             .await
             .map_err(|e| Self::error(ActionErrorKind::Open(fstab_path.to_path_buf(), e)))?;
 
-        // Make sure it doesn't already exist before we write to it.
         let mut fstab_buf = String::new();
         fstab
             .read_to_string(&mut fstab_buf)
             .await
             .map_err(|e| Self::error(ActionErrorKind::Read(fstab_path.to_owned(), e)))?;
 
-        let updated_buf = match existing_entry {
-            ExistingFstabEntry::NixInstallerEntry => {
-                // Update the entry
-                let mut current_fstab_lines = fstab_buf
-                    .lines()
-                    .map(|v| v.to_owned())
-                    .collect::<Vec<String>>();
-                let mut updated_line = false;
-                let mut saw_prelude = false;
-                let prelude = fstab_prelude_comment(apfs_volume_label);
-                for line in current_fstab_lines.iter_mut() {
-                    if line == &prelude {
-                        saw_prelude = true;
-                        continue;
-                    }
-                    if saw_prelude && line.split(&[' ', '\t']).nth(1) == Some("/nix") {
-                        *line = fstab_entry(&uuid);
-                        updated_line = true;
-                        break;
-                    }
+        let mut line_present = false;
+        let mut current_fstab_lines = fstab_buf
+            .lines()
+            .filter_map(|line| {
+                // Delete nix-installer entries with a "prelude" comment
+                if line.starts_with("# nix-installer") {
```

Comment:
Let's use more of the old prelude comment
```suggestion
                if line.starts_with("# nix-installer created volume labelled") {
```

Reply:
Done!

Reply:
Looks like you did it and then un-did it? :laughing:

Reply:
d'oh